numpy/linalg/lapack_lite/make_lite.py

```diff
@@ -276,7 +276,7 @@ def ensure_executable(name):
     try:
         which(name)
     except:
-        raise SystemExit(name + ' not found')
+        raise SystemExit(name + ' not found') from None
```

Comment:
This one should probably be `from e` too, although frankly I'd prefer to not touch this file in this PR.

Reply:
```suggestion
        raise SystemExit(name + ' not found')
```